force-app/main/default/customMetadata/Bucket.Absent.md-meta.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>Absent</label>
+    <protected>false</protected>
```

Comment:
The developerName is automatically disabled when created by a managed package. Not protecting it will allow the customers to use the buckets for other things if they so shall desire.